pom.xml

```diff
@@ -51,6 +51,11 @@
       <id>sonatype-nexus-staging</id>
       <url>https://s01.oss.sonatype.org/service/local/staging/deploy/maven2</url>
     </repository>
+    <repository>
+      <id>github</id>
+      <name>GitHub Packages</name>
+      <url>https://maven.pkg.github.com/madsop-nav/openhtmltopdf</url>
+    </repository>
```

Comment:
Maven only allows a single <repository> to be specified.

Reply:
We can use a profile block to override the repository when a github deployment flag is set.
This post describes the general pattern: https://stackoverflow.com/questions/16340673/setting-up-maven-to-allow-deployment-to-different-repositories-easily

Reply:
JReleaser supports deploying to multiple locations during a single execution 😏